.travis.yml

```diff
@@ -17,6 +17,8 @@ gemfile:
 
 matrix:
   exclude:
+    - rvm: 2.2.3
+      gemfile: Gemfile.rails5
```

Comment:
last time there is some build issue so someone from your team suggested the same but now I update it you can check